docs/content/querying/lookups.md

```diff
@@ -0,0 +1,191 @@
+|`valueColumn`|The name of the column containing the value|no|The second column|
+
+*example input*
+```
+bar,something,foo
+bat,something2,baz
+truck,something3,buck
+```
+
+*example lookupParseSpec*
+```json
+"lookupParseSpec": {
+  "type": "csv",
+  "columns": ["value","somethingElse","key"],
+  "keyColumn": "key",
+  "valueColumn": "value"
+}
+```
+
+### tsv lookupParseSpec
+
+|Parameter|Description|Required|Default|
+|---------|-----------|--------|-------|
+|`columns`|The list of columns in the csv file|yes|`null`|
+|`keyColumn`|The name of the column containing the key|no|The first column|
+|`valueColumn`|The name of the column containing the value|no|The second column|
+|`delimiter`|The delimiter in the file|no|tab (`\t`)|
+
+
+*example input*
+```
+bar|something,1|foo
+bat|something,2|baz
+truck|something,3|buck
+```
+
+*example lookupParseSpec*
+```json
+"lookupParseSpec": {
+  "type": "tsv",
+  "columns": ["value","somethingElse","key"],
+  "keyColumn": "key",
+  "valueColumn": "value",
+  "delimiter": "|"
+}
+```
+
+### customJson lookupParseSpec
+
+|Parameter|Description|Required|Default|
+|---------|-----------|--------|-------|
+|`keyFieldName`|The field name of the key|yes|null|
+|`valueFieldName`|The field name of the value|yes|null|
+
+*example input*
+```json
+{"key": "foo", "value": "bar", "somethingElse" : "something"}
+{"key": "baz", "value": "bat", "somethingElse" : "something"}
+{"key": "buck", "somethingElse": "something", "value": "truck"}
+```
+
+*example lookupParseSpec*
+```json
+"parseSpec": {
+  "type": "customJson",
+  "keyFieldName": "key",
+  "valueFieldName": "value"
+}
+```
+
+
+### simpleJson lookupParseSpec
+The `simpleJson` lookupParseSpec does not take any parameters. It is simply a line delimited json file where the field is the key, and the field's value is the value.
+
+*example input*
+ 
+```json
+{"foo": "bar"}
+{"baz": "bat"}
+{"buck": "truck"}
+```
+
+*example lookupParseSpec*
+```json
+"lookupParseSpec":{
+  "type": "simpleJson"
+}
+```
+
+
+## JDBC lookup update
+
+The JDBC lookups will pull from a database cache. If the `tsColumn` is set it must be able to accept comparisons in the format `'2015-01-01 00:00:00'`. For example, the following must be valid sql for the table `SELECT * FROM some_lookup_table WHERE timestamp_column >  '2015-01-01 00:00:00'`. If `tsColumn` is set, the caching service will attempt to only pull values that were written *after* the last sync. If `tsColumn` is not set, the entire table is pulled every time.
+
+```json
+{
+  "type":"jdbc",
+  "lookup":"some_lookup",
+  "connectorConfig":{
+    "createTables":true,
+    "connectURI":"jdbc:mysql://localhost:3306/druid",
+    "user":"druid",
+    "password":"diurd"
+  },
+  "table":"some_lookup_table",
+  "keyColumn":"the_old_dim_value",
+  "valueColumn":"the_new_dim_value",
+  "tsColumn":"timestamp_column",
+  "updateMs":600000
+}
+```
+
+# Kafka lookup update (Experimental)
```

Comment:
im wondering if we need to support the use case and perhaps that we can just release without it